library/sql.inc

```diff
@@ -650,4 +650,31 @@ function generic_sql_insert_id()
   return mysqli_insert_id($GLOBALS['dbh']);
 }
 
+
+/**
+ * Begin a Transaction.
+ */
+function sqlBeginTrans()
+{
+    $GLOBALS['adodb']['db']->BeginTrans();
+}
+
+
+/**
+ * Commit a transaction
+ */
+function sqlCommitTrans($ok=true)
```

Comment:
Its not used, but as sqlXXX functions map to AdoDB functions we made this be compatible with CommitTrans($ok) function ( See drivers/adodb-mysqli.inc.php ).